### PR TITLE
ci: skip docfx in main 'Kokoro' presubmit

### PR DESCRIPTION
**Pull request by busunkim96**
The `docfx` build is failing in continuous (reasons are unclear as we can't seem to reproduce the error locally). [log](https://source.cloud.google.com/results/invocations/8c0ad1bc-87fc-40ef-b730-484000fbd57e/targets/cloud-devrel%2Fclient-libraries%2Fpython%2Fgoogleapis%2Fpython-bigquery%2Fdocs%2Fdocs/log)

This PR makes the following changes:
* skip the `docfx` session in the main `Kokoro` presubmit by not listing it in the default sessions list
* use `NOX_SESSION` env var to run `docs` and `docfx` in `docs-presubmit` (needs an internal config change in 346339439)

After this is approved I will open a PR in synthtool to make the changes permanent.

### CODE REVIEW

**Review by tswast — APPROVED** [review]
Thanks!

File: .kokoro/build.sh [hunk 1]
```diff
@@ -41,5 +41,6 @@ python3.6 -m nox --version
 if [[ -n "${NOX_SESSION:-}" ]]; then
     python3.6 -m nox -s "${NOX_SESSION:-}"
 else
-    python3.6 -m nox
+    # docfx session runs in the 'docs-presubmit' presubmit
+    python3.6 -m nox -k "not docfx"
```
Comment (tswast) [inline .kokoro/build.sh:45]: I'm curious if we should be modifying the `noxfile` instead? I recall there's a way to mark certain sessions as "default"

**Review by tmatsuo — APPROVED** [review]

File: .kokoro/build.sh (hunk 1 above)
Comment (busunkim96) [reply .kokoro/build.sh]: Looks like [there is](https://nox.thea.codes/en/stable/config.html?highlight=default#modifying-nox-s-behavior-in-the-noxfile)! I modified the noxfile.py. PTAL

**Review by tswast — APPROVED** [review]

**Comment by busunkim96** [comment]
@tmatsuo I'm having trouble figuring out what's happening with trampoline. Could you take a look? https://source.cloud.google.com/results/invocations/2451449f-1926-423b-b139-5c318c6a2e30/targets/cloud-devrel%2Fclient-libraries%2Fpython%2Fgoogleapis%2Fpython-bigquery%2Fdocs%2Fdocs-presubmit/log

```
Digest: sha256:a0f4e7c15988bfb0524f25cb88635d0b9964d940a8a5c5bfbefc3ec651ed72cb
Status: Downloaded newer image for gcr.io/cloud-devrel-kokoro-resources/python-lib-docs:latest
================================================================
2020-12-08T17:40:13Z: Finished pulling the Docker image: gcr.io/cloud-devrel-kokoro-resources/python-lib-docs.
================================================================
================================================================
2020-12-08T17:40:13Z: Start building the docker image.
================================================================
================================================================
2020-12-08T17:40:17Z: Finished building the docker image.
================================================================
================================================================
2020-12-08T17:40:17Z: Running the tests in a Docker container.
================================================================
.kokoro/build.sh: line 18: cd: github/python-bigquery: No such file or directory
================================================================
2020-12-08T17:40:22Z: Build finished with 1
================================================================

```

**Comment by tmatsuo** [comment]
@busunkim96 

Maybe you can modify build.sh as follows:

```
if [[ -z "${PROJECT_ROOT:-}" ]]; then
    PROJECT_ROOT="github/python-bigquery"
fi

cd "${PROJECT_ROOT}"
```

**Review by tmatsuo — APPROVED** [review]

**Pull request merged**